### ISSUE TITLE
clarification on configuration

### ISSUE THREAD

**Issue by jj5836** [IngmarStein/unifi-sonos-doc#9]
I am trying to understand this configuration guidance:

> Multicast and Broadcast Control: off
> Devices -> $DEVICE -> Ports -> $AFFECTED_PORT

Does this refer to unify network devices and ports for hard-wired Sonos components or something else? Looking through the configuration options with the latest app I have not been able to find a matching configuration choice.

**Comment by WiggyToo** [comment]
I've assumed this only relates to the ports that any Sonos devices are hard wired to, accessed via port manager for the relevant switch (from memory).

Regards 
Gav

**Comment by IngmarStein** [comment]
Yes, @WiggyToo is correct. The UI has changed a bit over time and the current place is the port manager.

**Comment by jj5836** [comment]
Is it called "storm control" now? I don't see a setting "Multicast and Broadcast Control" in the port manager on the unifi web interface.

**Comment by proddy** [comment]
> Is it called "storm control" now? I don't see a setting "Multicast and Broadcast Control" in the port manager on the unifi web interface.

I see it under Settings->Wifi

<img width="319" alt="image" src="https://github.com/user-attachments/assets/9cd308c6-7ead-4a90-aa50-0828ea61ab13">

@IngmarStein thanks for your hint. I have removed my old Play:3 and now everything seems to work better now. I'll replace that old things with an Era 300.

**Comment by jj5836** [comment]
I do see that one also, but it's global for the WiFi network rather than a per-port thing as in the guide. 

The info text for "storm control" in the port manager does sounds similar to me at least:

<img width="343" alt="storm" src="https://github.com/user-attachments/assets/94f9f2de-a425-4976-a38f-8662c51467db">

**Comment by IngmarStein** [comment]
I've updated to doc to match the UI for Network Application 8.3.32, I hope that helps.
